cpp/src/aztec/honk/pcs/gemini/gemini.test.cpp

```diff
@@ -1,18 +1,104 @@
 #include "gemini.hpp"
 
 #include "../commitment_key.test.hpp"
+#include "polynomials/polynomial.hpp"
+#include <cstddef>
 #include <gtest/gtest.h>
+#include <gtest/internal/gtest-type-util.h>
```

Comment:
I don't think this private `include` is required.

Reply:
Removed